src/style.css

```diff
@@ -0,0 +1,99 @@
+:root {
+  --padding-left-big-search: 62px;
+  --padding-left-small-search: 33px;
+  --big-search-height: 72px;
+  --small-search-height: 42px;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url("./fonts/Avenir-Book.ttf");
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url("./fonts/Avenir-Heavy.ttf");
+  font-style: normal;
+  font-weight: 800;
+}
+
+.form {
+  position: relative;
+  height: 70px;
+  margin-bottom: 20px;
+}
+
+.forms-wrapper {
+  margin-top: 20px;
+}
+
+.label-search-icon {
+  position: absolute;
+  top: calc(var(--big-search-height) / 2);
+  left: calc(var(--padding-left-big-search) / 2);
+  transform: translateY(-50%) translateX(-50%);
+  cursor: text;
+}
+
+.icon {
+  width: 19px;
+  height: 19px;
+}
+
+.search {
+  border: 1px solid #e1e7ed;
+  width: 100%;
+  height: 100%;
+  border-radius: 4px;
+  padding-left: var(--padding-left-big-search);
+  font-weight: 300;
+  font-size: 16px;
+  color: #000;
+  font-family: Avenir, sans-serif;
+  margin-bottom: 20px;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+}
+
+.search:focus,
+.search:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.search:focus {
+  outline: none;
+  text-shadow: 0 4px 4px rgba(0, 0, 0, 0.25);
+  font-weight: 800;
+  background: linear-gradient(180deg, #fff, #f6f6f7);
+}
+
+.label-search-icon--small {
+  position: absolute;
+  left: calc(var(--padding-left-small-search) / 2);
+  top: calc(var(--small-search-height) / 2);
+  transform: translateY(-50%) translateX(-50%);
+  cursor: text;
+}
+
+.icon--small {
+  width: 10px;
+  height: 10px;
+}
+
+.search--small {
+  padding-left: var(--padding-left-small-search);
+  font-size: 14px;
+  background-size: 11px;
+  height: var(--small-search-height);
```

Comment:
To be consistent you should add height for a form elements. Add modifier to style form--small element and apply height there